compose/runtime/runtime/src/jvmMain/kotlin/androidx/compose/runtime/ActualJvm.jvm.kt

```diff
@@ -30,7 +30,7 @@ internal actual fun getCurrentThreadId(): Long = Thread.currentThread().id
 @InternalComposeApi
 actual fun identityHashCode(instance: Any?): Int = System.identityHashCode(instance)
 
-internal actual typealias TestOnly = org.jetbrains.annotations.TestOnly
+actual typealias TestOnly = org.jetbrains.annotations.TestOnly
```

Comment:
Actually it is not public now. When we compile it for JVM, it becomes internal. The code in `commonMain` dissapears

Reply:
But it accessible in commonMain.

Let's not overthink that is already done, and revert the fix only if decide making it internal in the future